package.json

```diff
@@ -38,12 +39,13 @@
   "engines": {
     "node": ">=8.0.0"
   },
+  "homepage": "https://github.com/mozilla/something-awesome/",
   "license": "MPL-2.0",
   "repository": "mozilla/something-awesome",
   "scripts": {
-    "bundle": "browserify frontend/src/main.js | uglifyjs > public/bundle.js",
+    "bundle": "browserify frontend/src/main.js | uglifyjs > public/bundle.js && node scripts/version",
```

Comment:
Not sure about this part. Since `bundle` gets run as part of the `prestart` hook, it means any existing /public/version.json file would be overwritten when you start the server. So not sure if I'd be killing a file on stage/prod.
Or if we should decouple the `node scripts/version` bit and have people run it manually for local/dev server.